src/docs/user/references/Notification-service.adoc

```diff
@@ -0,0 +1,237 @@
+| `TASK_IN_ERROR`
+| Task In_Error
+| ERROR
+| A task in the job encountered an error and has been set In-Error
+| A task in the job is In-Error
+| `TASK_WAITING_FOR_RESTART`
+| Task waiting on error
+| INFO
+| A task in the job is waiting for a new execution attempt after being in error, faulty or failed.
+| A task in the job is waiting for restart after an error
+| `TASK_RUNNING_TO_FINISHED`
+| Task finished
+| INFO
+| A task in the job is finished
+| A task in the job is finished
+| `TASK_IN_ERROR_TO_FINISHED`
+| In-Error Task finished
+| INFO
+| A task in the job was in error and is now finished
+| An In-Error task in the job is finished
+| `TASK_PENDING_TO_RUNNING`
+| Task running
+| INFO
+| A task in the job was pending and is now running
+| A task in the job is running
+| `TASK_SKIPPED`
+| Task skipped
+| INFO
+| The task was not executed, it was the non-selected branch of an IF/ELSE control flow action.
+| A task in the job is skipped
+| `TASK_REPLICATED`
+| Task replicated
+| INFO
+| A task in the job is replicated
+| A task in the job is replicated
+| `TASK_FINISHED_WITH_ERRORS`
+| Task finished with errors
+| ERROR
+| A task with no execution attempts left is faulty, failed or in error.
+| A task has finished with the an error status
+|===
+
+.Scheduler notifications
+[cols="2,2,2,5,5"]
+|===
+| *Type* | *Readable name* | *Severity* | *Message* | *Description*
+| `PAUSED`
+| Paused
+| WARNING
+| A user has paused the scheduler, all running jobs run to completion but jobs wont be submitted
+| The Scheduler is paused, all running jobs run to completion
+| `DB_DOWN`
+| Database down
+| CRITICAL
+| The Schedulers database is down
+| The Schedulers database is down
+| `KILLED`
+| Killed
+| WARNING
+| A user has killed the Scheduler, ProActive server needs to be restarted
+| The Scheduler is killed! The process of the scheduler's executable is killed. No interaction can be done anymore.
+| `RESUMED`
+| Resumed
+| INFO
+| The scheduler was paused or frozen and has been resumed
+| The Scheduler is resumed! Jobs can be submitted again
+| `STOPPED`
+| Stopped
+| WARNING
+| The scheduler has been stopped
+| The Scheduler is stopped! Jobs cannot be submitted anymore. Already running jobs run to completion, but not pending jobs.
+| `FROZEN`
+| Frozen
+| WARNING
+| The scheduler has been stopped
+| The Scheduler is frozen! All running tasks run to completion, but running jobs wait for the scheduler to resume.
+| `SHUTTING_DOWN`
+| Shutting down
+| WARNING
+| The scheduler has been shutdown
+| The Scheduler is shutting down... The scheduler is shutting down itself.
+| `STARTED`
+| Started
+| INFO
+| The scheduler has started
+| The Scheduler is started! Jobs can be submitted.
+|===
+
+[[_job_planner_notifications]]
+.Job Planner notifications (with example values)
+[cols="5,5"]
+|===
+|*Message* | *Description*
+| The calendar in the bucket was not take into account due to an exception
```

Comment:
The calendar in the bucket was not considered due to ..

Reply:
The calendar in the bucket was not taken into account ...
@mbenguig I think this is the message set by the code, alexandre cannot change it in the documentation.